cypress/integration/homework/classes/HomePage.js

```diff
@@ -0,0 +1,70 @@
+class HomePage {
+
+
+    clearButton = "#clearButton";
+    actionDropDown = "#selectOperationDropdown";
+    calculateButton = "#calculateButton";
+    numberField1 = "#number1Field";
+    numberField2 = "#number2Field";
+    integersOnlyCheckbox = "#integerSelect";
+
+
+    checkIfFieldsEnabled(field) {
+        cy.get(field).should('be.enabled')
+    }
+
+    enterFieldsWith(number1, number2, action) {
+        cy.get(this.numberField1).clear()
+        cy.get(this.numberField1).type(number1)
```

Comment:
You can combine 17-18 (also 19-20) lines into one line ;)
cy.get('selector).clear().type(number)
This would save some space